casper/src/test/scala/coop/rchain/casper/helper/RhoSpec.scala

```diff
@@ -67,43 +65,43 @@ class RhoSpec(
 
   def hasFailures(assertions: List[RhoTestAssertion]) = assertions.find(_.isSuccess).isDefined
 
-  private def testFrameworkContracts[F[_]: Log: Concurrent: Span](
-      testResultCollector: TestResultCollector[F]
-  ): Seq[SystemProcess.Definition[F]] = {
+  private def testFrameworkContracts(
```

Comment:
I don't understand these changes to `Task`. Why would you want to do that?

Reply:
The invocations already used Task. The call to runSync at the bottom of RhoSpec requires a concrete effect type.

Reply:
Ah, thanks. Now it makes more sense.
I think we could use `SyncIO` though. Then we don't have to commit to `Task`.